metadata-ingestion/src/datahub/ingestion/source/mode.py

```diff
@@ -1470,6 +1473,8 @@ def get_request():
                 response = self.session.get(
                     url, timeout=self.config.api_options.timeout
                 )
+                if response.status_code == 204:
```

Comment:
can we add a comment here? what's special about 204s?

Reply:
204 indicates "No Content", I think mode might be returning empty string in this case which isn't valid json. I can add a comment